src/main.rs

```diff
@@ -97,6 +99,12 @@ enum Commands {
         #[arg(short, long)]
         shell: Option<ShellEnum>,
     },
+    /// Generate shell completion script
+    Completions {
```

Comment:
```suggestion
    Completion {
```

I would prefer `pixi-pack completion`